package.json

```diff
@@ -24,6 +24,7 @@
         "@fastify/cors": "^8.4.1",
         "@fontsource/inter": "^5.0.15",
         "@headlessui/vue": "^1.7.16",
+        "@heroicons/vue": "^2.0.18",
```

Comment:
Please don't use heroicons, we use material design icons in the project https://pictogrammers.com/library/mdi/